### DISCUSSION TITLE
Yolov8 Pose Estimation on custom data error: AttributeError: 'PoseMetrics' object has no attribute 'curves_results'. See valid attributes below.

### DISCUSSION THREAD

**Discussion by palmcorp** [ultralytics/ultralytics#17060] [Q&A]
This error poped up after a training run on custom data. The legitmate attributes are: 
    Attributes:
        save_dir (Path): Path to the directory where the output plots should be saved.
        plot (bool): Whether to save the detection and segmentation plots.
        on_plot (func): An optional callback to pass plots path and data when they are rendered.
        names (list): List of class names.
        box (Metric): An instance of the Metric class to calculate box detection metrics.
        pose (Metric): An instance of the Metric class to calculate mask segmentation metrics.
        speed (dict): Dictionary to store the time taken in different phases of inference.
but the class documentation (https://docs.ultralytics.com/reference/utils/metrics/#ultralytics.utils.metrics.Metric.curves ) shows the following:
 attr curves_results property 
Who to bring this issue up with?

**Comment by glenn-jocher** [comment]
@palmcorp thank you for bringing this to our attention. Please open an issue on the GitHub repository with details of the error, and our team will look into it.
